src/harness/certs/generate_fake_certs.sh

```diff
@@ -64,7 +64,7 @@ openssl ca -cert root-ecc_ca.cert -keyfile private/root-ecc_ca.key -in sas-ecc_c
 echo "\n\nGenerate 'cbsd_ca' certificate/key"
 openssl req -new -newkey rsa:4096 -nodes \
     -reqexts cbsd_ca  -config ../../../cert/openssl.cnf \
-    -out cbsd_ca.csr -keyout private/cbsd_ca.key \
+    -out cbsd_ca.csr -keyout cbsd_ca.key \
```

Comment:
Why change this?

Reply:
The   change was done so that the private keys are not removed at the end of the   script. The private keys of the case are required for signing the short-lived   certificates.          The change has been reverted and the CA private keys are stored under the   'private' directory. Now the 'private' directory is not removed at the end of   the script. This is done to be consistent with the simple CRL server   implementation PR (this directory is also not deleted in this PR). For   certificate revocation and for creating short lived certificates this   directory is needed.